hackathon_site/hackathon_site/settings/base.py

```diff
@@ -57,6 +57,7 @@
     "applications",
     "event",
     "hardware",
+    "drf_yasg",
```

Comment:
Minor, I would put this above all our apps (so right between `dj_rest_auth` and `dashboard`). Just makes it a bit cleaner to pick out our apps from the list if they're all at the bottom.